protocol/communities_messenger_test.go

```diff
@@ -922,6 +922,7 @@ func (s *MessengerCommunitiesSuite) TestRequestAccess() {
 	response, err = s.bob.AcceptRequestToJoinCommunity(acceptRequestToJoin)
 	s.Require().NoError(err)
 	s.Require().NotNil(response)
+	s.Require().Len(response.RequestsToJoinCommunity()[0].RevealedAccounts, 1)
```

Comment:
Better to prevent panics in tests as much as possible
```suggestion
	s.Require().NotEmpty(response.RequestsToJoinCommunity())
	s.Require().Len(response.RequestsToJoinCommunity()[0].RevealedAccounts, 1)
```